readme.md

```diff
@@ -199,6 +199,19 @@ https://videopress.com/embed/*
 <iframe src="https://videopress.com/embed/zcnJVzQF" width="600" height="600"></iframe>
 ```
 
+### Microsoft Stream
+
+**Microsoft Stream urls**
+```
+https://web.microsoftstream.com/video/*
+https://web.microsoftstream.com/embed/video/*
+```
+
+**Microsoft Stream iframes**
+```
+<iframe src="https://web.microsoftstream.com/embed/video/73043e0c-cba8-482a-8a8e-0b72bc580ff4?autoplay=false&amp;showinfo=true" width="640" height="360"></iframe>
```

Comment:
To clean this up a bit, and follow the convention set by the other iframe examples, you could use something like this:
```suggestion
<iframe src="https://web.microsoftstream.com/embed/video/*?&" width="640" height="360"></iframe>
```